NYPLCardCreator/Classes/UserCredentialsViewController.swift

```diff
@@ -114,6 +114,8 @@ final class UserCredentialsViewController: TableViewController {
     
     self.tableView.estimatedRowHeight = 120
     self.tableView.allowsSelection = false
+    
+    NotificationCenter.default.addObserver(self, selector: #selector(self.endSignIn), name: Notification.Name.init("NYPLSettingsAccountsSignInFinishedNotification"), object: nil)
```

Comment:
For notification names, it is a good idea to declare a constant and always refer to it via that constant. Using a string directly leaves you open to very difficult to track down bugs in the event you ever have a typo (or someone accidentally deletes a character from the string later or some such).

My bigger concern here though is that this notification string is presumably being used because the constant is defined _in the host app_ and thus is not accessible from this library. Listening to notifications from the host app means this library no longer independently testable and creates invisible coupling.

Is some way we can rearchitect things so this library does not need to know about magic constants to listen to notifications generated in the host app?